backend/voting/admin.py

```diff
@@ -51,7 +51,7 @@ class Media:
 
 
 @admin.register(RankSubmission)
-class RankSubmissionAdmin(admin.ModelAdmin):
+class RankSubmissionAdmin(AdminUsersMixin):
```

Comment:
https://github.com/pythonitalia/pycon/blob/bced06dd103a41841904086cc5709e7aa70a36b4/backend/blog/admin.py#L31

you are missing the user_fk here, the mixin needs it to know where the user id is

```suggestion
class RankSubmissionAdmin(AdminUsersMixin):
    user_fk = 'speaker_id'
```